examples/vertex_mlops_enterprise/src/bqml_pipeline/src/pipeline.py

```diff
@@ -0,0 +1,216 @@
+    ):
+        """
+        Uploads the model to Vertex AI
+        Args:
+            project: Project ID
+            location: Region
+            bq_model_name: A fully-qualified model resource name or model ID.
+            Example: "projects/123/locations/us-central1/models/456" or
+            "456" when project and location are initialized or passed.
+        Returns:
+            None
+        """
+        from google.cloud import aiplatform as vertex_ai
+
+        model = vertex_ai.Model(model_name={MODEL_DISPLAY_NAME})
+
+        endpoint = vertex_ai.Endpoint.list(order_by="update_time")
+        endpoint = endpoint[-1]
+
+        model.deploy(
+            endpoint=endpoint,
+            min_replica_count=1,
+            max_replica_count=1,
+        )
+
+        model.wait()
+
+        return
+
+@dsl.pipeline(
+    name="creditcards-classifier-bqml-train",
+    description="Trains and deploys bqml model to detect fraud",
+    pipeline_root=PIPELINE_ROOT,
+    )
+def bqml_pipeline(
+        bq_table: str = BQ_INPUT_DATA,
+        model: str = MODEL_DISPLAY_NAME,
+        project: str = PROJECT_ID,
+        region: str = REGION,
+        endpoint_name: str = ENDPOINT_NAME,
+    ):
+
+        bq_model_op = BigqueryCreateModelJobOp(
+            project=project,
+            location=region,
+            query=f"""CREATE OR REPLACE MODEL `{BQ_DATASET_NAME}.{model}`
+            OPTIONS (
+                MODEL_TYPE='LOGISTIC_REG',
+                INPUT_LABEL_COLS=['{TARGET_COLUMN}'],
+                EARLY_STOP=TRUE,
+                model_registry='vertex_ai',
+                vertex_ai_model_id='{model}',
+                vertex_ai_model_version_aliases=['logit', 'experimental']
+            )
+            AS SELECT * EXCEPT(Time,ML_use) FROM `{bq_table}`            
+            """,
+        )
+
+        _ = BigqueryExplainPredictModelJobOp(
+            project=project,
+            location=region,
+            table_name=f"{bq_table}",
+            model=bq_model_op.outputs["model"],
+        )
+        
+        _ = BigqueryEvaluateModelJobOp(
+            project=project, location=region, model=bq_model_op.outputs["model"]
+        ).after(bq_model_op)
+
+#        get_evaluation_model_metrics_op = (
+#            get_model_evaluation_metrics(bq_evaluate_model_op.outputs["evaluation_metrics"])
+#            .after(bq_evaluate_model_op)
+#            .set_display_name("plot evaluation metrics")
+#        )
+
+#        # Check the model performance.
+#        with Condition(
+#            get_evaluation_model_metrics_op.outputs["accuracy"] > perf_threshold,
+#            name="accuracy is above threshold",
+#        ):
+
+#            endpoint_create_op = EndpointCreateOp(
+#                project=project, location=region, display_name=ENDPOINT_NAME
+#            )
+
+#            _ = upload_model_enpoint(
+#                project=project, location=region, bq_model_name=model
+#            ).after(endpoint_create_op)
```

Comment:
@javiergp is this something we want to keep?

Reply:
removed